components/brave_rewards/resources/extension/brave_rewards/components/panel.tsx

```diff
@@ -414,21 +420,41 @@ export class Panel extends React.Component<Props, State> {
     return defaultContribution
   }
 
+  initiateDelayCounter = () => {
+    this.delayTimer = setTimeout(() => {
+      clearTimeout(this.delayTimer)
```

Comment:
this should be on the first line in `initiateDelayCounter`. Calling clear here doesn't do anything as when you are inside here timer will be automatically cleared after state is set

Reply:
Done.